pkg/controller/compute/gke_test.go

```diff
@@ -39,6 +40,8 @@ import (
 	runtimev1alpha1 "github.com/crossplaneio/crossplane-runtime/apis/core/v1alpha1"
 	"github.com/crossplaneio/crossplane-runtime/pkg/test"
 
+	gcpcomputev1alpha2 "github.com/crossplaneio/stack-gcp/apis/compute/v1alpha2"
+
```

Comment:
Rogue whitespace here. We try to keep all the "local" (in this case `stack-gcp`) packages together in one block of imports.

Reply:
Done